lib/email.js

```diff
@@ -34,16 +45,16 @@ exports.sendConfirmationEmail = function(host, email, confirmation, userid, cb)
 exports.contactSeller = function(host, from, to, message, listingId, cb) {
   // setup e-mail data with unicode symbols
   var mailOptions = {
-      from: '"noreply" <noreply@tuskmarketplace.com>', // sender address
+      from: '"pleasereply" <pleasereply@tuskmarketplace.com>', // sender address
```

Comment:
What is it for? Because int his case they aren't even replying to pleasereply@tmp.com, but rather the email in replyto.